charts/scalardb-cluster/README.md

```diff
@@ -40,10 +39,10 @@ Current chart version is `1.0.0-SNAPSHOT`
 | scalardbCluster.image.repository | string | `"ghcr.io/scalar-labs/scalardb-cluster-node"` | Docker image reposiory of ScalarDB Cluster. |
 | scalardbCluster.image.tag | string | `""` | Override the image tag whose default is the chart appVersion |
 | scalardbCluster.imagePullSecrets | list | `[]` | Optionally specify an array of imagePullSecrets. Secrets must be manually created in the namespace. |
+| scalardbCluster.logLevel | string | `"INFO"` | The log level of ScalarDB Cluster |
 | scalardbCluster.nodeSelector | object | `{}` | nodeSelector is form of node selection constraint. |
 | scalardbCluster.podAnnotations | object | `{}` | Pod annotations for the scalardb-cluster deployment |
 | scalardbCluster.podSecurityContext | object | `{"seccompProfile":{"type":"RuntimeDefault"}}` | PodSecurityContext holds pod-level security attributes and common container settings. |
-| scalardbCluster.podSecurityPolicy.enabled | bool | `false` | Enable pod security policy |
```

Comment:
The `Pod Security Policies` is a deprecated (removed in the latest version) feature in Kubernetes. So, I removed this configuration.
(Note: I have already removed related resources when I create a temporary chart for testing ScalarDB Cluster.)